tests/Feature/TagControllerTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Models\Tag;
+use App\Models\User;
+use Database\Factories\TagFactory;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Spatie\Permission\Models\Role;
+use Tests\TestCase;
+
+
+class TagControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private User $user;
+
+    protected function setUp(): void
```

Comment:
I've uploaded this change for you. Check how you can use the `setUp()` method to preload some fixtures at the beginning of every test and clean some unnecessary code.

Reply:
Tnxs Jordi! I took a good look at what u did here! I'l implement this in the next tests!